test/integration/endpoints-test.js

```diff
@@ -0,0 +1,41 @@
+const { test } = require('tap')
+
+const UTIL = require('../util')
+const Cache = require('../../lib/cache')
+
+const [
+  gheVersion,
+  baseUrl,
+  cacheDir,
+  routesRoot,
+  routesDir
+] = [
+  UTIL.getGheVersion(),
+  UTIL.getBaseUrl(),
+  UTIL.getCacheDir(),
+  UTIL.getRoutesRoot(),
+  UTIL.getRoutesDir()
+]
```

Comment:
I think I would do not use the spread operator here, instead do

```js
const gheVersion = UTIL.getGheVersion()
```

It’s more readable, I think?